vision/src/autogluon/vision/image_prediction/image_prediction.py

```diff
@@ -246,17 +272,17 @@ def predict_feature(self, x):
             raise RuntimeError('Classifier is not initialized, try `fit` first.')
         return self._classifier.predict_feature(x)
 
-    def evaluate(self, val_data):
+    def evaluate(self, tuning_data):
```

Comment:
recommend it just be called `data`, since this isn't necessarily used for tuning.